core-redirects-studio-lib/src/main/java/com/tallence/core/redirects/studio/model/RedirectUpdateProperties.java

```diff
@@ -142,6 +143,8 @@ public Boolean getImported() {
     if (StringUtils.isNotEmpty(source)) {
       if (!sourceIsValid(source)) {
         errors.put(SOURCE, INVALID_SOURCE_VALUE);
+      } else if (sourceHasWhitespaces(source)) {
```

Comment:
Vielleicht könnte man das noch in den Test mit aufnehmen: https://github.com/tallence/core-redirects/blob/master/core-redirects-studio-lib/src/test/java/com/tallence/core/redirects/studio/model/RedirectUpdatePropertiesTest.java